### ISSUE TITLE
[Bug] No audio in any game on mac Ryujinx

### ISSUE THREAD

**Issue by m3dfatboi** [Ryujinx/Ryujinx#5594]
### Description of the issue

Trying to fix the smearing issue in RDR1 on mac, I updated Ryujinx to this version (https://github.com/Ryujinx/Ryujinx/pull/5589). But after that I was faced with another issue, now I don't have any audio in Ryujinx, like at all, in any game. I am kinda noob in all that simulators thing, so I need some help with that.

### Reproduction steps

Install Ryujinx Experimental GUI (Avalonia) Release-1.1.0+6921ef6 for mac
Launch any game
There is no audio

### Log file

00:00:00.039 |I| Configuration LogValueChange: ResScale set to: 1
00:00:00.041 |I| Configuration LogValueChange: ResScaleCustom set to: 1
00:00:00.041 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.041 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
00:00:00.042 |I| Configuration LogValueChange: BackendThreading set to: Auto
00:00:00.042 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
00:00:00.042 |I| Configuration LogValueChange: PreferredGpu set to: 0x106B_0xE0003EF
00:00:00.043 |I| Configuration LogValueChange: AntiAliasing set to: None
00:00:00.043 |I| Configuration LogValueChange: ScalingFilter set to: Bilinear
00:00:00.043 |I| Configuration LogValueChange: ScalingFilterLevel set to: 80
00:00:00.044 |I| Configuration LogValueChange: EnableDockedMode set to: True
00:00:00.048 |I| Configuration LogValueChange: EnableVsync set to: True
00:00:00.048 |I| Configuration LogValueChange: EnableShaderCache set to: True
00:00:00.048 |I| Configuration LogValueChange: EnableTextureRecompression set to: False
00:00:00.048 |I| Configuration LogValueChange: EnableMacroHLE set to: True
00:00:00.048 |I| Configuration LogValueChange: EnableColorSpacePassthrough set to: False
00:00:00.048 |I| Configuration LogValueChange: EnablePtc set to: True
00:00:00.048 |I| Configuration LogValueChange: EnableInternetAccess set to: False
00:00:00.048 |I| Configuration LogValueChange: EnableFsIntegrityChecks set to: True
00:00:00.048 |I| Configuration LogValueChange: FsGlobalAccessLogMode set to: 0
00:00:00.048 |I| Configuration LogValueChange: AudioBackend set to: OpenAl
00:00:00.048 |I| Configuration LogValueChange: AudioVolume set to: 1
00:00:00.049 |I| Configuration LogValueChange: MemoryManagerMode set to: HostMappedUnsafe
00:00:00.049 |I| Configuration LogValueChange: ExpandRam set to: False
00:00:00.049 |I| Configuration LogValueChange: IgnoreMissingServices set to: False
00:00:00.049 |I| Configuration LogValueChange: UseHypervisor set to: True
00:00:00.051 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.0+6921ef6
00:00:00.054 |N| Application Print: Operating System: Darwin 23.0.0 Darwin Kernel Version 23.0.0: Tue Aug  1 03:25:51 PDT 2023; root:xnu-10002.0.242.0.6~31/RELEASE_ARM64_T6000 (Arm64)
00:00:00.054 |N| Application Print: CPU: Apple M1 Pro� ; 8 logical
00:00:00.054 |N| Application Print: RAM: Total 16384 MiB ; Available 7371 MiB
00:00:00.056 |N| Application PrintSystemInfo: Logs Enabled: Info, Warning, Error, Guest, Stub
00:00:00.056 |N| Application PrintSystemInfo: Launch Mode: UserProfile
00:00:01.473 |E| .NET ThreadPool Worker 2023-08-19 08:59:19.596 Ryujinx[6422:435610] WARNING: Secure coding is not enabled for restorable state! Enable secure coding by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState: and returning YES.
00:00:00.108 |I| Gpu : Backend Threading (Auto): True
00:00:00.260 |N| Application LoadGuestApplication: Using Firmware Version: 16.0.3
00:00:00.260 |I| Application LoadGuestApplication: Loading as NSP.
00:00:00.278 |I| ModLoader QueryContentsDir: Searching mods for Title 01007820196A6000
00:00:00.280 |I| ModLoader QueryContentsDir: Searching mods for Title 01007820196A6000
00:00:00.282 |I| Loader Load: Loading rtld...
00:00:00.285 |I| Loader PrintRoSectionInfo: rtld:
    Module: nnrtld
00:00:00.286 |I| Loader Load: Loading main...
00:00:00.513 |I| Loader PrintRoSectionInfo: main:
    Module: t:\rdr2\src\sharpmake\build_vs2019\projects\rdr2\output\nx64\finald11\rdr_v83g-40550370_nx64_final_submission_gold_vanilla_ww.nss
00:00:00.514 |I| Loader Load: Loading sdk...
00:00:00.565 |I| Loader PrintRoSectionInfo: sdk:
    Module: nnSdk
    FS SDK Version: 15.3.2
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_libz-15_3_2-Release
                   SDK MW+Nintendo+NintendoSdk_nnSdk-15_3_2-Release
                   SDK MW+Nintendo+NintendoSdk_BuildTarget-64Bit
                   SDK MW+Nintendo+NintendoSDK_NVN-15_3_2-Release
00:00:00.597 |I| Loader LoadNsos: Loading image 0 at 0x0000000008500000...
00:00:00.598 |I| Loader LoadNsos: Loading image 1 at 0x0000000008504000...
00:00:00.724 |I| Loader LoadNsos: Loading image 2 at 0x0000000057695000...
00:00:00.727 |I| ModLoader LoadCheats: Build ids found for title 01007820196A6000:
    7501ABFE55FA41CFFEB46BD619BDCBF45B9CAE3A000000000000000000000000
    9515EF1F79EF3CB800BD85E5D3DE0D98F39381F3000000000000000000000000
    8618895246B420329B77746B960A66ABCBEDE992000000000000000000000000
00:00:00.729 |I| Application EnsureSaveData: Ensuring required savedata exists.
00:00:00.763 |I| Loader Start: Application Loaded: Red Dead Redemption v1.0.0 [01007820196a6000] [64-bit]
00:00:00.786 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:00.796 |E| .NET ThreadPool Worker [mvk-info] MoltenVK version 1.2.0, supporting Vulkan version 1.2.231.
00:00:00.797 |E| .NET ThreadPool Worker 	The following 83 Vulkan extensions are supported:
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_16bit_storage v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_8bit_storage v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_bind_memory2 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_buffer_device_address v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_create_renderpass2 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_dedicated_allocation v3
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_depth_stencil_resolve v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_descriptor_update_template v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_device_group v4
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_device_group_creation v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_driver_properties v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_dynamic_rendering v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_external_fence v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_external_fence_capabilities v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_external_memory v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_external_memory_capabilities v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_external_semaphore v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_external_semaphore_capabilities v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_fragment_shader_barycentric v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_get_memory_requirements2 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_get_physical_device_properties2 v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_get_surface_capabilities2 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_imageless_framebuffer v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_image_format_list v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_maintenance1 v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_maintenance2 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_maintenance3 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_multiview v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_portability_subset v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_push_descriptor v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_relaxed_block_layout v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_sampler_mirror_clamp_to_edge v3
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_sampler_ycbcr_conversion v14
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_separate_depth_stencil_layouts v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_shader_draw_parameters v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_shader_float_controls v4
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_shader_float16_int8 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_shader_subgroup_extended_types v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_spirv_1_4 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_storage_buffer_storage_class v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_surface v25
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_swapchain v70
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_swapchain_mutable_format v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_timeline_semaphore v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_uniform_buffer_standard_layout v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_variable_pointers v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_buffer_device_address v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_debug_marker v4
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_debug_report v10
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_debug_utils v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_descriptor_indexing v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_fragment_shader_interlock v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_hdr_metadata v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_host_query_reset v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_image_robustness v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_inline_uniform_block v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_memory_budget v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_metal_objects v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_metal_surface v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_post_depth_coverage v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_private_data v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_robustness2 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_sample_locations v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_scalar_block_layout v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_separate_stencil_usage v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_shader_stencil_export v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_shader_viewport_index_layer v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_subgroup_size_control v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_swapchain_colorspace v4
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_texel_buffer_alignment v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_texture_compression_astc_hdr v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_vertex_attribute_divisor v3
00:00:00.797 |E| .NET ThreadPool Worker 		VK_AMD_gpu_shader_half_float v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_AMD_negative_viewport_height v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_AMD_shader_image_load_store_lod v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_AMD_shader_trinary_minmax v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_IMG_format_pvrtc v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_INTEL_shader_integer_functions2 v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_GOOGLE_display_timing v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_MVK_macos_surface v3
00:00:00.797 |E| .NET ThreadPool Worker 		VK_MVK_moltenvk v36
00:00:00.797 |E| .NET ThreadPool Worker 		VK_NV_fragment_shader_barycentric v1
00:00:00.797 |E| .NET ThreadPool Worker 		VK_NV_glsl_shader v1
00:00:00.797 |E| .NET ThreadPool Worker [mvk-info] GPU device:
00:00:00.797 |E| .NET ThreadPool Worker 		model: Apple M1 Pro
00:00:00.797 |E| .NET ThreadPool Worker 		type: Integrated
00:00:00.797 |E| .NET ThreadPool Worker 		vendorID: 0x106b
00:00:00.797 |E| .NET ThreadPool Worker 		deviceID: 0xe0003ef
00:00:00.797 |E| .NET ThreadPool Worker 		pipelineCacheUUID: 260BAD42-0E00-03EF-0000-000000000000
00:00:00.797 |E| .NET ThreadPool Worker 	supports the following Metal Versions, GPU's and Feature Sets:
00:00:00.797 |E| .NET ThreadPool Worker 		Metal Shading Language 2.4
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Apple 7
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Apple 6
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Apple 5
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Apple 4
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Apple 3
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Apple 2
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Apple 1
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Mac 2
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Mac 1
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Common 3
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Common 2
00:00:00.797 |E| .NET ThreadPool Worker 		GPU Family Common 1
00:00:00.797 |E| .NET ThreadPool Worker 		macOS GPU Family 2 v1
00:00:00.797 |E| .NET ThreadPool Worker 		macOS GPU Family 1 v4
00:00:00.797 |E| .NET ThreadPool Worker 		macOS GPU Family 1 v3
00:00:00.797 |E| .NET ThreadPool Worker 		macOS GPU Family 1 v2
00:00:00.797 |E| .NET ThreadPool Worker 		macOS GPU Family 1 v1
00:00:00.797 |E| .NET ThreadPool Worker [mvk-info] Created VkInstance for Vulkan version 1.2.0, as requested by app, with the following 3 Vulkan extensions enabled:
00:00:00.797 |E| .NET ThreadPool Worker 		VK_KHR_surface v25
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_debug_utils v2
00:00:00.797 |E| .NET ThreadPool Worker 		VK_EXT_metal_surface v1
00:00:00.816 |E| .NET ThreadPool Worker [mvk-info] Vulkan semaphores using Metal implicit guarantees within a single queue.
00:00:00.816 |E| .NET ThreadPool Worker [mvk-info] Created VkDevice to run on GPU Apple M1 Pro with the following 8 Vulkan extensions enabled:
00:00:00.816 |E| .NET ThreadPool Worker 		VK_KHR_portability_subset v1
00:00:00.816 |E| .NET ThreadPool Worker 		VK_KHR_push_descriptor v2
00:00:00.816 |E| .NET ThreadPool Worker 		VK_KHR_shader_float16_int8 v1
00:00:00.816 |E| .NET ThreadPool Worker 		VK_KHR_swapchain v70
00:00:00.816 |E| .NET ThreadPool Worker 		VK_EXT_descriptor_indexing v2
00:00:00.816 |E| .NET ThreadPool Worker 		VK_EXT_fragment_shader_interlock v1
00:00:00.816 |E| .NET ThreadPool Worker 		VK_EXT_robustness2 v1
00:00:00.816 |E| .NET ThreadPool Worker 		VK_EXT_shader_stencil_export v1
00:00:00.864 |E| .NET ThreadPool Worker [mvk-info] Created 3 swapchain images with initial size (2560, 1458) and contents scale 2.0 for screen LG HDR 4K.
00:00:00.865 |N| GUI.RenderThread Gpu PrintGpuInformation: Apple Apple M1 Pro (Vulkan v1.2.231, Driver v0.2.2008)
00:00:00.870 |I| GPU.MainThread Gpu LoadShaders: Loading 352 shaders from the cache...
00:00:01.190 |S| HLE.OsThread.24 ServiceAm GetAppletResourceUserId: Stubbed. {appletResourceUserId: 1}
00:00:01.197 |S| HLE.OsThread.24 ServiceAm AcquireForegroundRights: Stubbed. 
00:00:01.197 |S| HLE.OsThread.24 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: False ; unknownFlag2: False ; unknownFlag3: True}
00:00:01.197 |S| HLE.OsThread.24 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:01.207 |S| HLE.OsThread.24 ServicePctl Initialize: Stubbed. 
00:00:01.535 |S| HLE.OsThread.24 ServiceNifm CreateRequest: Stubbed. {version: 2}
00:00:01.536 |S| HLE.OsThread.24 ServiceNifm Submit: Stubbed. 
00:00:01.536 |S| HLE.OsThread.24 ServiceNifm GetRequestState: Stubbed. 
00:00:01.536 |S| HLE.OsThread.24 ServiceNifm GetResult: Stubbed. 
00:00:01.540 |S| HLE.OsThread.5 ServiceBsd RegisterClient: Stubbed. 
00:00:01.541 |S| HLE.OsThread.5 ServiceBsd StartMonitoring: Stubbed. {unknown0: 0}
00:00:01.546 |S| HLE.OsThread.9 ServiceHid ActiveNpadImpl: Stubbed. {appletResourceUserId: 1 ; revision: 3}
00:00:01.549 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadIdType: Stubbed. 9 Players: Handheld,Player1,Player2,Player3,Player4,Player5,Player6,Player7,Player8
00:00:01.549 |S| HLE.OsThread.9 ServiceHid ActivateTouchScreen: Stubbed. {appletResourceUserId: 1}
00:00:01.549 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 100 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair}
00:00:01.567 |I| HLE.OsThread.6 AudioRenderer AcquireSessionId: Registered new renderer (0)
00:00:01.568 |I| HLE.OsThread.6 AudioRenderer Initialize: Initializing with REV11
00:00:01.580 |I| AudioProcessor.Worker AudioRenderer Work: Starting audio processor
00:00:01.580 |I| AudioRendererManager.Worker AudioRenderer SendCommands: Starting audio renderer
00:00:01.595 |I| HLE.OsThread.6 AudioRenderer Start: Starting renderer id 0
00:00:01.597 |S| HLE.OsThread.24 ServiceAcc InitializeApplicationInfo: Stubbed. {TitleId: 72189673298616320}
00:00:01.600 |S| HLE.OsThread.24 ServiceAcc IsUserAccountSwitchLocked: Stubbed. 
00:00:01.600 |S| HLE.OsThread.24 ServiceNs CheckAddOnContentMountStatus: Stubbed. 
00:00:01.601 |S| HLE.OsThread.24 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: True ; unknownFlag2: True ; unknownFlag3: False}
00:00:01.602 |S| HLE.OsThread.24 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:01.706 |S| HLE.OsThread.10 ServiceNv FinishInitialize: Stubbed. 
00:00:01.714 |S| HLE.OsThread.10 ServiceNv InitializeEx: Stubbed. 
00:00:01.724 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:00:01.760 |I| HLE.OsThread.24 SurfaceFlinger CreateLayerFromId: Creating layer 1
00:00:01.761 |I| HLE.OsThread.24 SurfaceFlinger CreateLayerFromId: Creating layer 2
00:00:01.801 |S| HLE.OsThread.10 ServiceNv SetNvMapFd: Stubbed. 
00:00:01.801 |S| HLE.OsThread.10 ServiceNv AllocGpfifoEx2: Stubbed. 
00:00:01.802 |S| HLE.OsThread.10 ServiceNv AllocObjCtx: Stubbed. 
00:00:01.803 |S| HLE.OsThread.10 ServiceNv SetErrorNotifier: Stubbed. 
00:00:01.803 |S| HLE.OsThread.10 ServiceNv SetUserData: Stubbed. 
00:00:01.804 |S| HLE.OsThread.10 ServiceNv SetTimeout: Stubbed. 
00:00:01.804 |S| HLE.OsThread.10 ServiceNv SetTimeslice: Stubbed. 
00:00:01.807 |S| HLE.OsThread.10 ServiceNv ZcullBind: Stubbed. 
00:00:01.809 |S| HLE.OsThread.24 ServiceAm SetOperationModeChangedNotification: Stubbed. {operationModeChangedNotification: True}
00:00:01.809 |S| HLE.OsThread.24 ServiceAm GetDefaultDisplayResolutionChangeEvent: Stubbed. 
00:00:02.403 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:02.787 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:03.327 |I| GPU.MainThread Gpu LoadShaders: Shader cache loaded.
00:00:03.405 |E| HLE.OsThread.10 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 3 took more than 1000ms, resuming execution...
00:00:03.448 |E| .NET ThreadPool Worker [mvk-info] Created 3 swapchain images with initial size (2560, 1484) and contents scale 2.0 for screen LG HDR 4K.
00:00:04.787 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:06.789 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:08.790 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:10.792 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:12.793 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:14.794 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:16.795 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:18.796 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:20.797 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:21.464 |I| HLE.OsThread.6 AudioRenderer AcquireSessionId: Registered new renderer (1)
00:00:21.464 |I| HLE.OsThread.6 AudioRenderer Initialize: Initializing with REV11
00:00:21.466 |I| HLE.OsThread.6 AudioRenderer Start: Starting renderer id 1
00:00:22.798 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:24.799 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:26.800 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:28.802 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:30.803 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:32.805 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:33.429 |E| HLE.OsThread.6 AudioRenderer Update: Unmap of memory pool (address: 0x14d43000, size 0xa000) failed!
00:00:33.941 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:34.455 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:34.807 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:36.808 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:38.809 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:40.810 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:41.811 |E| .NET ThreadPool Worker 2023-08-19 09:01:01.729 Ryujinx[6422:435610] ERROR: Can't have a toolbar in a window with <NSNextStepFrame: 0x165a592d0> as its `borderView`
00:00:42.811 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:42.955 |W| HLE.GuestThread.25 KernelSvc : WaitSynchronization(handleIndex: 0xFFFFFFFF) = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.23 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.26 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.33 KernelSvc : SendSyncRequest() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.27 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.35 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.28 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.37 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.42 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.29 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.955 |W| HLE.GuestThread.38 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.45 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.30 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.47 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.46 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.44 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.43 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.41 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.31 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.32 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.39 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.956 |W| HLE.GuestThread.40 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:42.958 |E| HLE.OsThread.13 SurfaceFlinger DequeueBuffer: No available buffer slots
00:00:42.959 |E| HLE.OsThread.13 SurfaceFlinger OnTransact: Error returned by transaction DequeueBuffer: Busy
00:00:42.964 |I| HLE.OsThread.6 AudioRenderer Stop: Stopping renderer id 0
00:00:42.964 |I| HLE.OsThread.6 AudioRenderer Stop: Stopped renderer id 0
00:00:42.964 |I| HLE.OsThread.6 AudioRenderer ReleaseSessionId: Unregistered renderer (0)
00:00:42.964 |I| HLE.OsThread.6 AudioRenderer Stop: Stopping renderer id 1
00:00:42.964 |I| HLE.OsThread.6 AudioRenderer Stop: Stopped renderer id 1
00:00:42.964 |I| HLE.OsThread.6 AudioRenderer ReleaseSessionId: Unregistered renderer (1)
00:00:42.965 |I| AudioProcessor.Worker AudioRenderer Work: Stopping audio processor
00:00:42.966 |I| HLE.OsThread.6 AudioRenderer StopLocked: Stopped audio renderer
00:00:42.969 |W| GUI.WindowThread ServiceNv Close: Closing channel

### OS

Sonoma Version 14.0 Beta (23A5312d)

### Ryujinx version

1.1.0+6921ef6

### List of applied mods

None

**Comment by makigumo** [comment]
Try the **SDL2** audio backend. All others don't seem to work on macOS.

**Comment by m3dfatboi** [comment]
> Try the **SDL2** audio backend. All others don't seem to work on macOS.

It worked. Thaks a lot!

**Comment by m3dfatboi** [comment]
> Try the **SDL2** audio backend. All others don't seem to work on macOS.

It worked. Thanks a lot!
